src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">AnySoftKeyboard - change_me Language Pack</string>
-    <string name="keyboard_name">change_me keyboard</string>
-    <string name="dictionary_name">change_me</string>
+    <string name="app_name">AnySoftKeyboard - Indonesian Language Pack</string>
+    <string name="keyboard_name">Indonesian keyboard</string>
```

Comment:
allright, thanks :)

Yes, If I don't change the value, the result will be like 
![this](http://i12.photobucket.com/albums/a209/azzamsa/photo_2017-07-11_20-20-41_zpsirlkmj6u.jpg)